tests/end-to-end/backup-restore-test/backupe2e/asset-store/testsuite/asset.go

```diff
@@ -0,0 +1,104 @@
+package testsuite
```

Comment:
Hmm, I've seen this code somewhere... could you please vendor the `asset-store` integration test and use the package? I know that you would need to make there some modifications (like exporting the type), but it's definitely worth it.

Reply:
Or - better - please move it to `commons`